README.md

```diff
@@ -4,18 +4,26 @@
 [![License](https://img.shields.io/github/license/vitruv-tools/Vitruv-CaseStudies.svg)](https://raw.githubusercontent.com/vitruv-tools/Vitruv-CaseStudies/main/LICENSE)
 
-[Vitruvius](https://vitruv.tools) is a framework for view-based software development. It assumes different models to be used for describing a software system,
-which are automatically kept consistent by the framework and its applications. For general information on Vitruvius, see our [GitHub Organisation](https://github.com/vitruv-tools) and our [Wiki](https://github.com/vitruv-tools/.github/wiki).
+[Vitruvius](https://vitruv.tools) is a framework for view-based (software) development.
+It assumes different models to be used for describing a system, which are automatically kept consistent by the framework executing (semi-)automated rules that preserve consistency.
+These models are modified only via views, which are projections from the underlying models.
+For general information on Vitruvius, see our [GitHub Organisation](https://github.com/vitruv-tools) and our [Wiki](https://github.com/vitruv-tools/.github/wiki).
 
-This project contains case studies, in particular an application of the Vitruvius framework in the domain of component-based systems engineering. It provides functionality for creating views on Java code, UML models and PCM models ([Palladio component models](https://github.com/palladiosimulator)) and for preserving consistency between such models. In addition, it provides test cases for demo applications based on the simple [families, persons and insurance metamodels](https://github.com/kit-sdq/DemoMetamodels) defined in the [DSLs repository](https://github.com/vitruv-tools/Vitruv-DSLs).
+This project contains case studies for the [Vitruvius framework](https://github.com/vitruv-tools/Vitruv), in particular an example application of Vitruvius in the domain of component-based systems engineering.
+It provides functionality for creating views on Java code, UML models and PCM models ([Palladio Component Model](https://github.com/palladiosimulator)) and for preserving consistency between these models by providing `ChangePropagationSpecifications`.
+In addition, it provides test cases for demo applications based on the simple [families, persons and insurance metamodels](https://github.com/kit-sdq/DemoMetamodels) defined in the [Vitruv-DSLs](https://github.com/vitruv-tools/Vitruv-DSLs) repository.
 
 ## Installation
 
-Vitruvius can be installed in Eclipse via the [nightly update site](https://vitruv.tools/updatesite/nightly). A wiki page provides [detailed instructions for using or extending Vitruvius](https://github.com/vitruv-tools/.github/wiki/Getting-Started).
+Vitruvius can be installed in Eclipse via the [nightly update site](https://vitruv.tools/updatesite/nightly). A wiki page provides [detailed instructions for using or extending Vitruvius or parts of it](https://github.com/vitruv-tools/.github/wiki/Getting-Started).
 
 ## Project Development
```

Comment:
Should we keep this part or remove it (as the development comments are the same for every repository and are detailed on the Getting-Started page)?

Reply:
I see, xtext is different. I would nevertheless argue to remove this section as it provides not enough detail to get started with the development.